app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+
+    if len(command_list) == 3 and command.startswith("cp "):
+        _, file_name_in, file_name_out = command_list
+
+        if file_name_in != file_name_out:
+            try:
+                with (open(file_name_in) as file_in,
+                      open(file_name_out, "w") as file_out):
+                    file_out.write(file_in.read())
+            except (FileNotFoundError, PermissionError) as e:
+                raise e
```

Comment:
Now lets think, do you actually need this try block if we are not handling exceptions at all, and just calling em?))

Reply:
It's better to use str(e). It is not guaranteed across all exception types that the exception will have message attribute